RadarSDK/Include/RadarGeofence.h

```diff
@@ -41,7 +41,7 @@
 @property (nullable, copy, nonatomic, readonly) NSDictionary *metadata;
 
 /**
- The geometry of the geofence.
+ The geometry of the geofence, which can be cast to either `RadarCircleGeometry` or `RadarPolygonGeometry`.
```

Comment:
do we want to pull common properties like `center` / `radius` into the `RadarGeofenceGeometry` interface, so it's more explicit around why we're using the superclass pattern?